gallery/templates/view_file.html

```diff
@@ -10,6 +10,22 @@
 {% block body %}
 
 <div class="container">
+    {% if file.hidden or lockdown %}
+        <div class="row">
+            {% if lockdown %}
+                <div class="alert alert-danger">
+                    <p class="mb-0">Gallery is in Lockdown! Regular members cannot view content.</p>
```

Comment:
My one nitpick is the wording of `Regular members` to refer to non-eboard/non-RTP members. Maybe just `Members cannot view content.`, as it's probably assumed that eboard/RTPs control lockdown.